integration-tests/src/test/java/org/zowe/apiml/integration/authentication/schemes/ZoweJwtSchemeTest.java

```diff
@@ -56,6 +56,7 @@ void givenCorrectClientCertificateInRequest() {
             .get(URL)
             .then()
             .body("headers.cookie", startsWith("apimlAuthenticationToken"))
+            .body("headers.authorization", startsWith("Bearer"))
```

Comment:
given the code has logic to handle authorization headers starting with Bearer, maybe we could have a test with Basic authorization header, to validate the behaviour